internal/sandbox/binary.go

```
@@ -85,6 +92,280 @@ func GetPostgresLibDir(binDir, version string) string {
+	}
+
+	// Create container (not started)
+	createCmd := exec.CommandContext(ctx, "docker", "create", "--name", containerName, svc.Image)
+	if out, err := createCmd.CombinedOutput(); err != nil {
+		return fmt.Errorf("docker create %s failed: %s: %w", svc.Image, string(out), err)
+	}
+
+	// Ensure cleanup
+	defer func() {
+		rmCmd := exec.Command("docker", "rm", containerName)
+		rmCmd.Run()
```

golangci-lint (2.5.0)

[error] Error return value of `rmCmd.Run` is not checked

(errcheck)